Articles/Packages/Extending/creating-a-package.md

```diff
@@ -0,0 +1,141 @@
+
+### The Package Content section
+
+This section is used to determine which things the package should contain. We will fill in the following things:
+
+| Property | Value | Note |
+| -------- | ----- | ---- |
+| Content | _Empty_ | Here you can include content - fx if you want to create a starter kit. Not relevant for this package though |
+| Document Types | _Empty_ | Similar to the Content picker above. Important to note that if you include content you will need to also pick all its dependencies in this and the next steps for it to be packaged together! |
+| Templates | _Empty_ | See `Document Types` above |
+| Stylesheets | _Empty_ | This will come from the wwwroot/css folder. If you have stylesheets you want to include from other locations you can do so at a later step |
+| Macros | _Empty_ | See `Document Types` above |
+| Languages | _Empty_ | See `Document Types` above - all text is hardcoded or within our own lang folder in this package, so this is not needed |
+| Dictionary | _Empty_ | See `Document Types` above |
+| Data Types | _Empty_ | See `Document Types` above |
+
+### The Package Files section
+
+In this section you will be able to select all of your own custom files. We will start with the `Path to file` option.
+
+Since everything in our Dashboard is from the same folder in `App_Plugins`, we can select the folder and it will include all items inside that folder:
+
+![Selecting the files you want in your package](images/select-files-for-package.png)
+
+We will leave the `Package options view` selector empty, but in case you were wondering you can select an HTML file here that will show up as package options. Here is an example from the package [Articulate](https://github.com/Shazwazza/Articulate/tree/master/src/Articulate.Web/App_Plugins/Articulate/BackOffice/PackageOptions):
+
+![Gif of Articulates use of Package options](images/package-options.gif)
+
+### The Package Actions section
+
+Here you can add package actions. There are a number of [default package actions](./package-actions.md) and you can also create your own [custom package actions](./custom-package-actions.md)
+
+Finally after filling out all the info we can select `Create` to create the package. We will download it, in order to take a closer look at what it contains in the generated zip file.
+
+## Inspecting a package zip
+
+When you download and then open the zip package you will find that it looks like this:
+
+![Content of a zip package](images/zip-package.png)
+
+The 5 highlighted files were the ones the package contained that we created in the [Creating a Custom Dashboard tutorial](../../../Tutorials/Creating-a-Custom-Dashboard/index.md), however there is another file here called `package.xml` - so let's take a look at that - it looks like this:
```

Comment:
```suggestion
The 5 highlighted files contained in the page package are the ones that we created in the [Creating a Custom Dashboard tutorial](../../../Tutorials/Creating-a-Custom-Dashboard/index.md).  However there is another file here called `package.xml` - so let's take a look at that. It looks like this:
```

Reply:
"page package" sounds a bit odd to me, wouldn't just "contained in the package" be better?

Reply:
It would yes, sorry, that's what I meant - not sure where "page" came from.

Reply:
No problem I have to admit that I just copy/pasted the text and made sure that it wasn't incorrect. So great corrections overall :D

Reply:
Sure, I realise this has come from existing docs... just started looking and thought would give it a proof read too.